rdkit/Chem/Draw/UnitTestDraw.py

```diff
@@ -7,18 +7,15 @@
 #  which is included in the file license.txt, found at the root
 #  of the RDKit source tree.
 #
-import re
 import os
 import pathlib
+import re
 import sys
 import tempfile
 import unittest
 
 from rdkit import Chem
-from rdkit.Chem import AllChem
-from rdkit.Chem import Draw
-from rdkit.Chem import rdDepictor
-from rdkit.Chem import rdMolDescriptors
+from rdkit.Chem import AllChem, Draw, rdDepictor, rdMolDescriptors
```

Comment:
This change is not connected to this PR and violates the RDKit's normal coding style (it's also incompatible with PEP8)

Reply:
Oops, Black autoformatter did that despite my trying to prevent it.